docs/operator-docs/provider.md

```diff
@@ -0,0 +1,77 @@
+---
+id: provider
+title: Provider Configuration
+hide_title: true
+---
+
+# Provider Configuration
+
+In Cumulus a Provider represents a endpoint from which data is ingested.   For example, a http server serving data from a data provider, or an S3 bucket deployed within your organization with data staged for ingest into Cumulus.
+
+Please note:
+
+* While *connection* configuration is defined here, things that are more specific to a specific ingest setup (e.g. 'What target directory should we be pulling from' or 'How is duplicate handling configured?') are generally defined in a Rule or Collection, not the Provider.
+* Task-specific provider configurations that are expected to be configured on a *per-workflow* basis exist, and may be tweaked to allow for task-specific configuration (e.g. https timeout values for the `discover-granules` task).
```

Comment:
I'm not really sure what this is trying to say

Reply:
Ouch.  

This is trying to end-run Operators who come in looking to tweak connection behaviors that aren't intended for the Provider scope.    For example, we have several things like http/https timeouts that are *task* configurations that you might expect to be Provider/global oriented. 


How about 
```suggestion
* Task-specific provider configurations that are expected to be configured on a *per-workflow* basis exist, these are generally configured as part of the workflow definition and are documented in task documentation/README).
```

Reply:
```suggestion
* There is some provider behavior which is controlled by task-specific configuration and not the provider definition. This configuration has to be set on a **per-workflow** basis. For example, see the [`httpListTimeout` configuration on the `discover-granules` task](https://github.com/nasa/cumulus/blob/master/tasks/discover-granules/schemas/config.json#L84)
```

Reply:
👍  I can live with that.